packages/graphql-language-service-interface/src/GraphQLLanguageService.js

```diff
@@ -158,18 +163,24 @@ export class GraphQLLanguageService {
       fragmentDefinitions,
     );
 
-    const localFragDefinitions = ast.definitions
-      .filter(definition => definition.kind === FRAGMENT_DEFINITION)
-      .map(definition => ({
-        file: filePath,
+    const localFragDefinitions = ast.definitions.filter(
+      definition => definition.kind === FRAGMENT_DEFINITION,
+    );
+
+    const typeCastedDefs = ((localFragDefinitions: any): Array<FragmentDefinitionNode>);
```

Comment:
Not sure if there is a clever way to do this with `%checks` and a predicate function but this seems reasonable for now anyway.